lib/repl.js

```diff
@@ -1156,9 +1156,9 @@ function complete(line, callback) {
     }
 
     completionGroupsLoaded();
-  } else if (match = line.match(/fs\.\s*[a-z][a-zA-Z]+\(\s*["'](.*)/)) {
+  } else if (match = line.match(/fs(\.promises)?\.\s*[a-z][a-zA-Z]+\(\s*["'](.*)/)) {
```

Comment:
```suggestion
  } else if (match = line.match(/fs(?:\.promises)?\.\s*[a-z][a-zA-Z]+\(\s*["'`](.*)/)) {
```

This way it does not include the `.promises` part in the matches and will also match backticks as quote.

The linter does not complain about the line length due to being a regular expression but the line becomes relatively long as well, thus, I would move the expression to the top level.